### DISCUSSION TITLE
Welcome to filament-users Discussions!

### DISCUSSION THREAD

**Discussion by fadymondy** [tomatophp/filament-users#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fabiobettiol** [comment]
Hi!
Is there a way to define roles to teams, instead of users? (set the permissions on a per-team basis)
Thnx!

**Comment by tmindiashvili1** [comment]
Hello! Just wondering if you have a timeline or planned release for Filament v5 support. Appreciate your work on this package
